app/scrapers/hessen_scraper.rb

```diff
@@ -0,0 +1,183 @@
+    leg, ref = extract_reference(block)
+    {
+      legislative_term: leg,
+      full_reference: ref,
+      reference: [leg, ref].join('/'),
+      doctype: extract_interpellation_type(block),
+      title: extract_title(block),
+      is_answer: true,
+    }
+  end
+
+  def self.extract_result_from_search(page)
+    page.search('//tbody[@name="RecordRepeatStart"]').first
+  end
+
+=begin
+
+  weird format:
+
+  KlAnfr    Weiß, Marius, SPD; Schmitt, Norbert, SPD; Decker,
+            Wolfgang, SPD; Geis, Kerstin, SPD; Hofmeyer,
+            Brigitte, SPD; Kummer, Gerald, SPD; Löber,
+            Angelika, SPD; Warnecke, Torsten, SPD
+            19.11.2014 und Antw 17.02.2015 Drs "
+
+  will be brought into firstname lastname (party), ... format
+=end
+
+  def self.extract_originators(text)
+    text = text.sub('KlAnfr', '').sub('GrAnfr', '')
+    lines = text.split("\n")
+    first_date_matched = false
+    lines = lines.map do |line|
+      # we can ignore everything after a line with a date
+      first_date_matched = true if !get_matches_for_date_pattern(line).nil?
+      line.gsub(/\p{Z}+/, ' ').strip
+      line = nil if first_date_matched
+      line
+    end
+    lines = lines.reject(&:nil?).join(' ').split(';').map do |s|
+      sa = s.split(',')
+      party = sa.pop
+      name = sa.reverse.join ' '
+      name + ' (' + party + '), ' unless party.include?('Fraktion')
+    end
+    NamePartyExtractor.new(lines.join).extract
+  end
+
+  def self.extraxct_answer_line(text)
+    text.split("\n").each do |s|
+      s = s.gsub(/\p{Z}+/, ' ').strip
+      return s if (s.include?('Antw') || s.include?('und Antw'))
+    end
+    return nil
+  end
+
+  def self.get_matches_for_date_pattern(line)
+    /\d{2}\.\d{2}\.\d{4}/.match line
+  end
+
+  def self.get_date_from_detail_line(line)
+    matches = get_matches_for_date_pattern(line)
+    return nil if matches.nil?
+    Date.parse(matches[(matches.length-1)])
+  end
+
+  class Overview < Scraper
+    SEARCH_URL = BASE_URL + '/starweb/LIS/servlet.starweb?path=LIS/PdPi_FL.web'
+
+    def supports_streaming?
+      true
+    end
+
+    def scrape
+      search_url = SEARCH_URL+ '&wp=WP'+ @legislative_term.to_s +
+        '&search=((UTYP1=GR+OR+KL)+AND+ANTWEIN+NOT+(!+OR+%22%22))'
+      streaming = block_given?
+      m = mechanize
+      mp = m.get search_url
+
+      blocks = HessenScraper.extract_blocks(mp)
+      papers = []
+      blocks.each do |block|
+        begin
+          paper = HessenScraper.extract_paper(block)
+        rescue => e
+          logger.warn e
+          next
+        end
+        next if paper.nil?
+        if streaming
+          yield paper
+        else
+          papers << paper
+        end
+      end
+      papers unless streaming
+    end
+  end
+
+
+  class Detail < DetailScraper
+    SEARCH_URL = BASE_URL + '/starweb/LIS/Pd_Eingang.htm'
+
+    def scrape
+      m = mechanize
+      mp = m.get SEARCH_URL
+      mp = m.click(mp.link_with(:text=>'Suche'))
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.